app/graphql/types/category.rb

```diff
@@ -0,0 +1,50 @@
+class Types::Category < Types::BaseObject
+  description 'Information about a specific Category'
+
+  field :id, ID, null: false
+
+  field :title, String,
+    null: false,
+    description: 'The name of the category.'
+
+  field :description, String,
+    null: true,
+    description: 'A description of the category.'
+
+  field :slug, String,
+    null: false,
+    description: 'The URL-friendly identifier of this Category.'
+
+  field :anidb_id, Integer,
+    null: true,
+    description: 'Kitsu category mapping to Anidb category.'
+
+  field :total_media_count, Integer,
+    null: false,
+    description: 'The amount of media that include this category.'
+
+  field :nsfw, Boolean,
+    null: false,
+    description: 'Whether the category is Not-Safe-for-Work.',
+    method: :nsfw?
+
+  field :image, Types::Image,
+    null: true,
+    description: 'An image of the category.'
```

Comment:
Lets take this out since its not used

Reply:
should we take it out or deprecate it?

Reply:
Mark it as deprecated in the JSON:API docs and don't include it here.